<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>950</width>
     <height>500</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>10</y>
      <width>10</width>
      <height>480</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>센서 상태</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableSensor">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>441</width>
      <height>401</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Index</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sensor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Value</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>시스템 상태</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHealth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>로봇</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>시스템 실시간 상세 상태</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>출고구역</string>
    </property>
   </widget>
   <widget class="QFrame" name="robot_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>RGB센서 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>RGB센서 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>서보모터 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>스탭모터 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>DC모터   1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="robot_rgb1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="robot_rgb2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="robot_serbo1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="robot_step1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="robot_dc1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>R 구역</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Y 구역</string>
    </property>
   </widget>
   <widget class="QFrame" name="r_status">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>근접센서 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>스탭모터 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="r_proxi1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="r_step1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="g_status">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>근접센서 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>스탭모터 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="g_proxi1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="g_step1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>G 구역</string>
    </property>
   </widget>
   <widget class="QFrame" name="y_status">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>근접센서 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>스탭모터 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="y_proxi1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="y_step1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="sh_status">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>RGB센서 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sh_rgb1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>41</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
